simulator/src/main/java/com/hedera/block/simulator/generator/GeneratorInjectionModule.java

```diff
@@ -39,6 +39,8 @@ static BlockStreamManager providesBlockStreamManager(BlockStreamConfig config) {
 
         if ("BlockAsDirBlockStreamManager".equalsIgnoreCase(config.managerImplementation())) {
             return new BlockAsDirBlockStreamManager(config);
+        } else if ("BlockAsFileLargeDataSets".equalsIgnoreCase(config.managerImplementation())) {
+            return new BlockAsFileLargeDataSets(config);
```

Comment:
We might want to refactor this, for readibility and performance in the future, if the items get too much. switch is faster if more than 5 items, but that's just a note for later.

Reply:
yes I agree that this can be further improved in the future, maybe even an Enum with all the current implementations to keep it clean. Thanks for the suggestion, we can keep it in mind.

Reply:
Maybe in the future 👍

Reply:
Don't think performance is a factor here, more readability and simplicity.

Reply:
You can use strings in a switch statement now, and this is a good place for that.
Not really worthwhile for only 2 options, however.